.github/workflows/release.yaml

```
@@ -333,9 +333,9 @@ jobs:
           if-no-files-found: error
           path: target/${{ matrix.target}}/release/libxayn_ai_ffi_c_${{ matrix.target }}.a
 
-  build-wasm-lib-release:
-    name: build-wasm-lib-release
-    needs: registry-cache
+  build-release-wasm-lib:
```

should we already use the plural here (and across all ci files)? we probably have to support `wasm64` next to the current `wasm32` in the near future anyways, then the ci naming could stay the same. this would also align the naming with the other platform libs job names.
```suggestion
  build-release-wasm-libs:
```

---

I think using wasm64 is only required if we need more than 4 gb of ram, but I can change the name to be consistent with the other jobs